python/paddle/v2/fluid/data_feeder.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import print_function
+from framework import Variable
+import core
+import layers
+import numpy
+import six.moves as six
+
+
+class DataToLoDTensorConverter(object):
+    def __init__(self, place, lod_level, shape, batch_size_dim, dtype):
```

Comment:
It seems that `batch_size_dim` here is no use.

Reply:
Yes. batch_size_dim could be removed.